### PR TITLE
Links fix

### PR DESCRIPTION
**Pull request by nikunj1705**
broken links fixed.

### CODE REVIEW

File: docs/bot-builder/basic/general-nodes.md [hunk 1]
```diff
@@ -82,8 +82,6 @@ Because General Nodes are very simple in terms of features (no entities/api call
 
 To simplify the process of creating these FAQ General nodes, there is an option to upload a TSV file (Tab separated file).
 
-Here is a [sample TSV file](https://haptik-stagingcf.haptikapi.com/general_nodes_faq.tsv) which can be used to create FAQ based general nodes.
```
Comment (GanadiniAkshay) [inline docs/bot-builder/basic/general-nodes.md]: why are we removing this?

File: docs/agent-chat/meta.json [hunk 2]
```diff
@@ -16,8 +16,7 @@
             "action-toolbars",
             "smart-actions",
             "businesses",
-            "teams",
-            "analytics"
```
Comment (GanadiniAkshay) [inline docs/agent-chat/meta.json]: why are we removing analytics?

File: docs/bot-builder/basic/general-nodes.md (hunk 1 above)
Comment (nikunj1705) [reply docs/bot-builder/basic/general-nodes.md]: link is wrong.

File: docs/agent-chat/meta.json (hunk 2 above)
Comment (nikunj1705) [reply docs/agent-chat/meta.json]: the file is not there yet. The next button on Teams page gives a 404. Person adding analytics should add in meta.json again.

**Pull request merged**